Dockerfile.test

```diff
@@ -10,13 +10,12 @@ RUN apk add --no-cache libssh2 libssh2-dev iputils
 
 # Add trusted CAs for communicating with external services
 RUN apk update && apk add --no-cache ca-certificates tzdata && update-ca-certificates
+RUN apk add --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main
```

Comment:
what's happening here?

Reply:
`apk` fails to find the repository by its name (probably a search error) but if it pulls directly from the link this does not produce an error

this is the log for the error prior to the current successful build

https://travis-ci.com/github/PlaceOS/core/builds/191362399#L234-L241

Reply:
you're not actually adding `watchexec`